l10n_es_partner_mercantil/__manifest__.py

```diff
@@ -5,7 +5,7 @@
 {
     "name": "Partner Mercantil",
     "summary": u"Añade los datos del registro mercantil a la empresa",
-    "version": "9.0.1.0.0",
+    "version": "10.0.1.0.0",
```

Comment:
You do not need to enter "10.0". See "Version numbers" in: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md